R/do_adjustRtime-functions.R

```diff
@@ -582,3 +582,418 @@ adjustRtimeSubset <- function(rtraw, rtadj, subset,
     }
     rtadj
 }
+
+###########################################################
+###### LamaParama
+
+#' @title Landmark-based alignment: aligning a dataset against an external
+#' reference
+#'
+#' @aliases LamaParama-class
+#'
+#' @description
+#' Alignment is achieved using the ['adjustRtime()'] method with a `param` of
+#' class `LamaParama`. This method #' corrects retention time by aligning
+#' chromatographic data with an external reference dataset. Peaks in the
+#' experimental chromatographic data are aligned to predefined landmark
+#' features called "lamas" based on their mass-to-charge ratio (m/z) and
+#' retention time.
+#'
+#' This alignment algorithm matches chromatographic peaks from the experimental
+#' data to the lamas, fitting a model based on this match to adjust their
+#' retention times and minimize discrepancies. This adjustment is performed
+#' file by file. Adjustable parameters such as `ppm`, `tolerance`, and
+#' `toleranceRt` define acceptable deviations during the matching process.
+#' It's crucial to note that only lamas and chromatographic peaks exhibiting a
+#' one-to-one mapping are considered when estimating retention time shifts. If
+#' a file has no peaks matching with lamas, no adjustment will be performed,
+#' and the file will be returned as-is. Users can evaluate this matching, for
```

Comment:
I would rather say *the retention times will be returned as-is* instead of *the file will be returned as-is*.